lib/directories.zsh

```diff
@@ -24,10 +24,11 @@ alias rd=rmdir
 alias d='dirs -v | head -10'
 
 # List directory contents
-alias lsa='ls -lah'
-alias l='ls -lah'
+# alias lsa='ls -lah'
+alias l='ls'
```

Comment:
`ls` is the abbreviation of *list*, and `l` is again the abbreviation of `ls`.
then, `la` indicates `ls -a`, `ll` indicates `ls -l`, and `lla` (or `lal`) indicates `ls -l -a`.

This derivation looks sane. 😄